### PR TITLE
PR12899-follow-up edits to mod content

### PR DESCRIPTION
**Pull request by bmcelvee**
Follow-up to https://github.com/openshift/openshift-docs/pull/12899

### CODE REVIEW

**Comment by bmcelvee** [comment]
We define a “working directory” but don’t tell the user where the “working directory” is--Is it in a POD/Host?

Need to address.

**Comment by bmcelvee** [comment]
> We define a “working directory” but don’t tell the user where the “working directory” is--Is it in a POD/Host?
> 
> Need to address.

Defined in "Adding input secrets and ConfigMaps"?

File: builds/creating-build-inputs.adoc [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 :context: creating-build-inputs
 [id='creating-build-inputs_{context}']
+include::modules/common-attributes.adoc[]
```
Comment (bmcelvee) [inline builds/creating-build-inputs.adoc]: This doesn't quite work here? Where should it be located in the assembly module?

Comment (bmcelvee) [reply builds/creating-build-inputs.adoc]: ^ fixed

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :snowflake: 

(You can ignore my comments, they're content related and just reminders for myself.)

File: modules/builds-source-secret-ssh-key-auth.adoc [hunk 2]
```diff
@@ -6,35 +6,30 @@
...
 default.
 
-. Prerequisites
+.Prerequisites
 
 * SSH key based authentication requires a private SSH key.
 
 .Procedure
 
+Two files are created: the public key and a corresponding private key (one of
```
Comment (kalexand-rh) [inline modules/builds-source-secret-ssh-key-auth.adoc]: I think there's another step hidden in here.

**Review by kalexand-rh — COMMENTED** [review]
Just one comment. If it's building clean, it LGTM.

File: modules/builds-source-secret-ssh-key-auth.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/builds-source-secret-ssh-key-auth.adoc]: Thanks! I'll update in just a bit :smiley_cat:

**Pull request merged**